### DISCUSSION TITLE
Adding more data to the same project

### DISCUSSION THREAD

**Discussion by WaelMI** [asreview/asreview#1671] [Q&A]
Hello, 

I have a question regarding using ASReview for live data streaming and was hoping that anyone could help. 

I have a specific use case where I want to connect ASReview to a data stream, through which I can keep adding data to the same project and keep using the already-trained model. 
In my understanding this is possible, but I don't know how to do it. 
Can anyone help? 

Thank you,
Wael

**Comment by bianwenjun** [comment]
maybe you can add add papers from several datasets into one reference manager such as zotero.
 and then you can output all of them in the ris format through zotero.

hope it will help you :)

**Comment by Rensvandeschoot** [comment]
Would the compose functionality of[ ASReview-datatools](https://github.com/asreview/asreview-datatools?tab=readme-ov-file#data-compose-experimental) be helpful? 
Moreover, the model is re-trained at every iteration of the active learning cycle, as is explained in this paper: https://osf.io/preprints/psyarxiv/g93zf. This might also make your challenge more manageable.
